pkg/drivers/common.go

```diff
@@ -122,6 +125,21 @@ func MakeDiskImage(d *drivers.BaseDriver, boot2dockerURL string, diskSize int) e
 	return nil
 }
 
+// KillAPIServerProc will kill an api server proc if it exists
+func KillAPIServerProc(runner command.Runner) error {
+	// first check if it exists
+	rr, err := runner.RunCmd(exec.Command("pgrep", "kube-apiserver"))
+	if err == nil { // this means we might have a running kube-apiserver
+		pid, err := strconv.Atoi(rr.Stdout.String())
```

Comment:
IMHO, it should be considered exceptional and show an error if there is an apiserver running after the containers have been removed.

Reply:
on containerd I killed the container and I saw there was something still listening the api port with lsof command. this command only runs if the proc exists, If you agree I could take it out for none and leave it only for kic.
(for none as there might be their own kube-apiserver that we dont wanna kill) 

what do u think ?